apps/api/src/app/widgets/dtos/feeds-response.dto.ts

```diff
@@ -209,6 +209,14 @@ export class NotificationFeedItemDto implements INotificationDto {
   })
   payload?: Record<string, unknown>;
 
+  @ApiPropertyOptional({
+    description: 'The data sent with the notification.',
+    type: 'object',
+    nullable: true,
+    example: { key: 'value' },
+  })
```

Comment:
```suggestion
  @ApiPropertyOptional({
    description: 'The data sent with the notification.',
    type: 'object',
    nullable: true,
    example: { key: 'value' },
    additionalProperties: true
  })
```

this is needed for proper SDK type generation

Reply:
Fixed